docs/reusing-typecheck-results.md

```diff
@@ -0,0 +1,251 @@
+|Ending Optimizations                |  9,0015|  0,0000|    216|      0|      0|      0|    457|     45|
+|Encoding OptData                    |  9,0090|  0,0010|    216|      0|      0|      0|    457|     45|
+|TailCall Checks                     |  9,0190|  0,0013|    216|      0|      0|      0|    457|     45|
+// main4, main5
+|TAST -> IL                          |  9,0463|  0,0210|    217|      0|      0|      0|    458|     46|
+|>Write Started                      |  9,0586|  0,0018|    217|      0|      0|      0|    457|     46|
+|>Module Generation Preparation      |  9,0655|  0,0001|    217|      0|      0|      0|    457|     46|
+|>Module Generation Pass 1           |  9,0719|  0,0005|    217|      0|      0|      0|    457|     46|
+|>Module Generation Pass 2           |  9,0820|  0,0031|    218|      0|      0|      0|    457|     46|
+|>Module Generation Pass 3           |  9,0894|  0,0005|    218|      0|      0|      0|    457|     46|
+|>Module Generation Pass 4           |  9,0968|  0,0002|    218|      0|      0|      0|    457|     46|
+|>Finalize Module Generation Results |  9,1032|  0,0003|    218|      0|      0|      0|    457|     46|
+|>Generated Tables and Code          |  9,1120|  0,0005|    218|      0|      0|      0|    457|     46|
+|>Layout Header of Tables            |  9,1189|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Build String/Blob Address Tables   |  9,1276|  0,0006|    218|      0|      0|      0|    457|     46|
+|>Sort Tables                        |  9,1339|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Write Header of tablebuf           |  9,1430|  0,0001|    218|      0|      0|      0|    457|     46|
+|>Write Tables to tablebuf           |  9,1503|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Layout Metadata                    |  9,1588|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Write Metadata Header              |  9,1656|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Write Metadata Tables              |  9,1731|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Write Metadata Strings             |  9,1819|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Write Metadata User Strings        |  9,1897|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Write Blob Stream                  |  9,1976|  0,0000|    218|      0|      0|      0|    457|     46|
+|>Fixup Metadata                     |  9,2033|  0,0002|    218|      0|      0|      0|    457|     46|
+|>Generated IL and metadata          |  9,2102|  0,0015|    218|      0|      0|      0|    457|     46|
+|>Layout image                       |  9,2195|  0,0025|    218|      0|      0|      0|    457|     46|
+|>Writing Image                      |  9,2272|  0,0006|    218|      0|      0|      0|    456|     46|
+|>Signing Image                      |  9,2338|  0,0000|    218|      0|      0|      0|    456|     46|
```

Comment:
```suggestion

```

All the steps starting with a pipe are just children of writing .NET Binary, they are not helpful in this doc as they stand.

Reply:
Fair, I just copypasted --times, this is a redundant part indeed.